contracts/rootchain/RootChain.sol

```diff
@@ -21,6 +21,7 @@ contract RootChain is Ownable {
     event Deposit(address depositor, uint256 amount);
     event FinalizedExit(uint priority, address owner, uint256 amount);
     event AddedToBalances(address owner, uint256 amount);
+    event QueueSize(uint256 size);
```

Comment:
Why do we need this? Is it just for testing?

Reply:
lol yes, please remove it

Reply:
oops i added a few events while i was testing and forgot to get rid of them. good catch